features/step_definitions/user_steps.rb

```diff
@@ -304,6 +304,12 @@ def get_series_name(age, classname, name)
   step %{I should get confirmation that I changed my username}
 end
 
+When(/^I am logged in as "([^"]*)" at "([^"]*)"$/) do |login, time|
```

Comment:
I don't think we need a new step for this. We can do `time is frozen at`, then `I am logged in as`, then `jump in our Delorean`.